app/controllers/admin/users_controller.rb

```diff
@@ -46,7 +52,16 @@ def update
   private
   def set_selected_menu_items
     @main_selected = :customers
-    @sub_nav_selected = :users
+    
+    if params[:role] && params[:role] == "admin"
```

Comment:
```suggestion
    if params[:role] == "admin"
```
I think if `params[:role]` is `nil`, this would just fail, so you wouldn't have to check if the key is present.